src/rand_impls.rs

```diff
@@ -176,6 +207,94 @@ impl Rand for bool {
     }
 }
 
+impl Rand for Ordering {
+    #[inline]
+    fn rand<R: Rng>(rng: &mut R) -> Ordering {
+        match rng.gen_range(0, 3) {
+            0 => Ordering::Less,
+            1 => Ordering::Equal,
+            2 => Ordering::Greater,
```

Comment:
1/3 chance of each ordering may be okay, but it's arbitrary and even weird. Not really sure; it's not *worse* than any other option but may still not be what's wanted. Still, we already have `Option`..

Reply:
Fair! I was considering this for the sake of testing, and IMHO each should be equally likely to be tested.